data-products/src/recommendation_api/sql/recommended_by_recency.sql

```diff
@@ -58,4 +58,5 @@ SELECT
     TOPIC,
     PUBLISHER
 FROM deduped
+WHERE TOPIC is not null
```

Comment:
Might as well apply this to `publisher`?
```suggestion
WHERE TOPIC is not null
AND PUBLISHER is not null
```